classes/decision/steps/Email.php

```diff
@@ -57,6 +58,17 @@ public function canChangeRecipients(bool $value): self
         return $this;
     }
 
+    /**
+     * Should the recipient names be shown in the
+     * email body and subject when writing the email
+     */
+    public function anonymizeRecipients(bool $value): self
```

Comment:
I think we can start using `static` instead of `self` in such cases, then an extension of this class would reference itself instead of the parent class.

Reply:
Oh neat. I thought the `static` return type was only for static methods, but this is really useful. We should make add this to the code style in our contributor's guide.

I'm going to leave this as-is, but I agree it'd be a good refactor project to update uses of `self`.